app/src/main/java/com/android/sample/model/ingredient/networkData/OpenFoodFactsIngredientRepository.kt

```diff
@@ -117,7 +119,6 @@ class OpenFoodFactsIngredientRepository(private val client: OkHttpClient) : Ingr
     val request =
         Request.Builder()
             .url(url)
-            // TODO: Add a proper User-Agent
```

Comment:
Nice catch! That TODO has been around so long, it might as well predate the Earth itself! 😂 Time to give it a proper send-off.